scripts/tests/sanitycheck/test_sanitycheck.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+This test file contains foundational testcases for Sanitycheck tool
+"""
+
+import os
+import imp
+import sys
+import pytest
+ZEPHYR_BASE = os.getenv("ZEPHYR_BASE")
+sys.path.insert(0, os.path.join(ZEPHYR_BASE, "scripts/"))
+from sanity_chk import scl
+sanitycheck = imp.load_source('sanitycheck', ZEPHYR_BASE + '/scripts/sanitycheck')
+
+@pytest.fixture(name='test_data')
+def _test_data():
+    """ Pytest fixture to set the path of test_data"""
+    data = ZEPHYR_BASE + "/scripts/tests/sanitycheck/test_data/"
+    return data
+
+def test_zephyr_base_set():
```

Comment:
this test is kinda useless, because your test script depends on ZEPHYR_BASE already and would not work if it is not set..